tests/platform_tests/cli/test_show_platform.py

```diff
@@ -99,6 +99,9 @@ def test_show_platform_psustatus(duthosts, rand_one_dut_hostname):
     @summary: Verify output of `show platform psustatus`
     """
     duthost = duthosts[rand_one_dut_hostname]
+    logging.info("Check pmon daemon status")
+    assert check_pmon_daemon_status(duthost), "Not all pmon daemons running."
```

Comment:
Do we need an import for this method?

Reply:
Method check_pmon_daemon_status imported

Reply:
I manually ran the code an got an exception.
```
def test_show_platform_psustatus(duthosts, rand_one_dut_hostname):
        """
        @summary: Verify output of `show platform psustatus`
        """
        duthost = duthosts[rand_one_dut_hostname]
        logging.info("Check pmon daemon status")
>       assert check_pmon_daemon_status(duthost), "Not all pmon daemons running."
E       NameError: global name 'check_pmon_daemon_status' is not defined
```
Did I miss something?